core/src/test/java/org/infinispan/persistence/PassivationFunctionalTest.java

```diff
@@ -154,30 +165,30 @@ public void testRemoveAndReplace() throws PersistenceException {
       assertNotInCacheAndStore("k1");
 
       assert cache.put("k2", "v2-NEW").equals("v2");
-      assertInCacheNotInStore("k2", "v2-NEW");
+      assertInCacheAndInStore("k2", "v2-NEW");
```

Comment:
I am a bit confused. Shouldn't the update occurr only on cache and not in store?